api/v1alpha1/dockermachine_types.go

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
If nothing else about the new filesystem structure, it's great to see the API changes show up prior to the generated crds :joy: